src/py/flwr/server/superlink/fleet/grpc_rere/fleet_servicer.py

```diff
@@ -51,27 +51,29 @@ def CreateNode(
         self, request: CreateNodeRequest, context: grpc.ServicerContext
     ) -> CreateNodeResponse:
         """."""
-        log(INFO, "FleetServicer.CreateNode")
+        log(INFO, "[Fleet.CreateNode] Request %s", request)
```

Comment:
We don't want to log the full request on log level `INFO`

I liked the previous approach more: have a short log message with log level `INFO`, and an additional more verbose one with log level `DEBUG`